src/index.html

```diff
@@ -8,6 +8,28 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div class="product-card" data-qa="card">
+    <div class="image product-card__image"></div>
+    <p class="product-card__description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
```

Comment:
```suggestion
    <h3 class="product-card__description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
```
Це, швидше, заголовок